### PR TITLE
chore: various add-ons: remove use of setParent

### PR DESCRIPTION
**Pull request by kingthorin**
In follow-up to: https://github.com/zaproxy/zaproxy/pull/6598

- automation
- imagelocationscanner
- reports
- retire
- saml
- soap

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

**Comment by kingthorin** [comment]
I see the typo in the commit msg, will tweak after build passes.

**Comment by thc202** [comment]
My bad, Wappalyzer can't be changed (maybe there are others?).

**Comment by kingthorin** [comment]
Will tweak it.

**Comment by kingthorin** [comment]
This should be good now.

**Review by psiinon — APPROVED** [review]

File: addOns/automation/CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
-
+### Changed
+- Maintenance changes.
```
Comment (thc202) [inline addOns/automation/CHANGELOG.md:8]: Not really needed, just affects test code (same for reports).

Comment (kingthorin) [reply addOns/automation/CHANGELOG.md:8]: Ok, will tweak.

File: addOns/imagelocationscanner/CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,8 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
-
+### CHanged
```
Comment (thc202) [inline addOns/imagelocationscanner/CHANGELOG.md:7]: The case needs to be tweaked.

**Comment by kingthorin** [comment]
Done

**Comment by thc202** [comment]
Thank you!

**Pull request merged**